### ISSUE TITLE
phi0978.phi001.perseus-lat2.xml hyphenation question

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/canonical-latinLit#147]
In looking at this file, words appear to be split as in print. I believe from legacy issues and other work that we should be combining the words, yes?

**Comment by lcerrato** [comment]
does anyone else have any thoughts on this?

**Comment by PonteIneptique** [comment]
Yes we should be combining :) Poetry is a bit of a unknown though...

**Comment by lcerrato** [comment]
Thanks! @PonteIneptique
@annettegessner, you might want to double check the data entry standards and/or see if there is a transformation available to handle new work files.
